mobile/src/libs/enums/navigation/root-screen-name.enum.ts

```diff
@@ -1,6 +1,8 @@
 const RootScreenName = {
-  SIGN_UP: 'Sign Up',
+  HOME: 'Home',
+  MAIN: 'Main Screens',
```

Comment:
Home should be inside MainScreenName not RootScreenName
```suggestion
  MAIN: 'Main',
```